invenio_app_ils/mappings/v6/documents/document-v1.0.0.json

```diff
@@ -39,6 +39,26 @@
             }
           }
         },
+        "series": {
```

Comment:
the `series_pids` are missing from the mapping

Reply:
Do you think it's necessary to have it when the `series` object has the same information?